src/.nuget/packages.config

```diff
@@ -1,7 +1,7 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
   <package id="FSharp.Compiler.Tools" version="4.0.1.20" />
-  <package id="Google.Protobuf.Tools" version="3.0.0" />
+  <package id="Google.Protobuf.Tools" version="3.2.0" />
```

Comment:
There is a `GenerateProtos.cmd` that may no longer be linked, but also needs updating to this 3.2.0 version if we truly require it